### DISCUSSION TITLE
Without TSCH/6Tisch, node level time synchronisation in Cooja

### DISCUSSION THREAD

**Discussion by LeoLee0097** [contiki-ng/contiki-ng#1720] [Q&A] [answered]
Dear all
I am working with z1 motes (MSP430) with NullNet examples in the Cooja simulator, and I would like to know are all my motes synchronized at the ms level in Cooja? 
I set the same send interval for all of them (6 nodes, for instance), and I hope they could send messages simultaneously (roughly) so that I could calculate the end-to-end delay. I followed this advice in the following link, but I have no idea if it ok or not.
Current solution - https://contiki-developers.narkive.com/jRJheA43/time-synchronization-in-contiki
Could you please give me your valuable advice? I would so much appreciate your help.

**Comment by LeoLee0097** [comment]
BTW, now I am using _badis djamaa_'s advice in https://contiki-developers.narkive.com/jRJheA43/time-synchronization-in-contiki and set the `<motedelay_us>0</motedelay_us>` in my cooja_project.css

**Comment by atiselsts** [comment, marked as answer]
Starting them all at the same time is definitely the simplest approach. It if works for you, just do it.
